figma-tokens.json

```diff
@@ -345,6 +353,40 @@
         "value": "9999",
         "type": "borderRadii"
       }
+    },
+    "borderRadius": {
```

Comment:
I'm wondering if it is possible to rename this borderRadius to borderRadii, we should not have both borderReadius and borderRadii